corehq/apps/domain/calculations.py

```diff
@@ -119,6 +119,23 @@ def forms(domain, *args):
     return get_number_of_forms_in_domain(domain)
 
 
+def forms_in_last(domain, days):
+    """
+    Returns the number of forms submitted in the last given number of days
+    """
+    now = datetime.utcnow()
+    then = json_format_datetime(now - timedelta(days=int(days)))
+    now = json_format_datetime(now)
+    q = {
+        'query': {
+            'range': {
+                'received_on': {'from': then, 'to': now}},
+            'filter': {
+                'and': ADD_TO_ES_FILTER['forms'][:]}}}
+    data = es_query(params={'domain.exact': domain}, q=q, es_index='forms', size=1)
+    return data['hits']['total'] if data.get('hits') else 0
```

Comment:
Would be better to use https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/forms.py

`FormES().domain(domain).submitted(gte=then)`